lib/reciter.rb

```diff
@@ -15,4 +15,43 @@ def read
     "This is the horse and the hound and the horn that belonged to the farmer sowing his corn that kept the rooster that crowed in the morn that woke the priest all shaven and shorn that married the man all tattered and torn that kissed the maiden all forlorn that milked the cow with the crumpled horn that tossed the dog that worried the cat that killed the rat that ate the malt that lay in the house that Jack built.\n"
   end
 
+  def recite_random_rhyme(random_indices=generate_random_indices)
```

Comment:
This is a great way to make methods unit-testable.

I think it might be more conventional to do the default param as `random_indices=nil`
and then first line of method `random_indices ||= generate_random_indices`